frontend/config/environment.js

```diff
@@ -53,7 +53,6 @@ module.exports = function(environment) {
     environment,
     rootURL: '/',
     locationType: 'auto',
-    historySupportMiddleware: true,
```

Comment:
what was this for?

Reply:
This was necessary for getting the addon to work, but I removed it once I removed the addon.